### PR TITLE
[ACR-1655] Add descripton and features of new sensors

### PR DESCRIPTION
**Pull request by lavataste**
ACR-1655 : https://code.sec.samsung.net/jira/browse/ACR-1655

Add descripton and features of new sensors 

- SENSOR_ORIENTATION : orientation sensor based on rotation_vector
- SENSOR_GYROSCOPE_ORIENTATION : orientation sensor based on gyroscope_rotation_vector
- SENSOR_GEOMAGNETIC_ORIENTATION: orientation sensor based on geomagnetic_rotation_vector

### CODE REVIEW

File: docs/application/native/guides/location-sensors/device-sensors.md [hunk 1]
```diff
@@ -315,6 +318,25 @@ The following table provides information about the accelerometer output for a de
...
+- Pitch
+- Roll
+
+The following table lists the measurement data that the geomagnetic orientation sensor provides:
+
+**Table: Measurement data detected by the geomagnetic orientation sensor**
+
+| Measurement        | Type                 | Range                                  | Unit         |
+|--------------------|----------------------|----------------------------------------|--------------|
+| Timestamp          | `unsigned long long` | -                                      | Microseconds |
```
[the remark this answers is not in the document]
Comment (lavataste) [reply docs/application/native/guides/location-sensors/device-sensors.md]: done

File: docs/application/dotnet/guides/sensors/device-sensors.md [hunk 2]
```diff
@@ -266,6 +291,30 @@ The following table lists the measurement data that the gyroscope provides.
 | Y           | `float`              | Min. value = -573.0<br>Max. value = 573.0 | Degrees/s (&deg;/s) |
 | Z           | `float`              | Min. value = -573.0<br>Max. value = 573.0 | Degrees/s (&deg;/s) |
 
+ <a name="gyro_orientation"></a>
+
+## Gyroscope Orientation Sensor
+
+The gyroscope orientation sensor combines the 3-axis accelerometer and 3-axis gyroscope to determine the orientation (rotation angles) of the device. The gyroscope orientation sensor is similar to the orientation sensor, but it does not use a geomagnetic field. Thus, azimuth represents a relative rotation angle, not an absolute reference. The gyroscope orientation is the output of a software/hardware-based sensor fusion solution that uses the accelerometer and gyroscope. The gyroscope orientation sensor output is an alternative representation to the gyroscope rotation vector sensor output used to determine the rotation of the device, and it is calculated in terms of Euler angles:
+
+-   Azimuth
```
[the remark this answers is not in the document]
Comment (lavataste) [reply docs/application/dotnet/guides/sensors/device-sensors.md]: done

**Review by hyotaekshim — APPROVED** [review]
Looks good to me

**Comment by lavataste** [comment]
@arisha-srbd

**Comment by lavataste** [comment]
@arisha-srbd 
Thanks for your review
Most of your suggestions have been applied.

Changing "Azimuth" to "Yaw" is only in "gyroscope" orientation so, words "Azimuth" in other orientation sensor remain same.

**Review by mijin85cho — COMMENTED** [review]
Hello, @arisha-srbd 

I saw your questions and below are my answers & some comments I'd like to share.


> We have noticed that the font and page layout format is different in comparison to the other contents. @mijin85cho we would appreciate your input on this.

It is correct. Poland applied new style to .NET guides. So .NET guides have different style.

> Line 9: “The main features of the Sensor API include: “
Suggestion: I am not sure why ‘Sensor’ has been capitalized here?

Global : API names are  common compound nouns. So capitalized one is correct.

**Reference :** https://github.com/Samsung/tizen-docs/blob/master/styleguide/naming-rules.md#proper-nouns


> @mijin85cho We have noticed that the dependency versions mentioned are different for the .net file and the native file. 
We are concerned whether this is supposed to be the same for both the files or is it okay the way it is?

.NET was added later than Native or Web. As .NET has been added since Tizen 4.0, dependency must be different if the guide is about an old module that included in Native or Web before Tizen 4.0.

**Comment by lavataste** [comment]
@mijin85cho

**Comment by lavataste** [comment]
@mijin85cho 
Thanks for your review.

Isn't the "used" is for 'geomagnetic rotation vector sensor output", not "geomagnetic orientation sensor output"?
-> for rotation vector, so I did rollback the expressions "and is used" to "used" 

1. In this page, there are two tables, each for mobile and wearable.
If a feature is supported in both mobile and wearable,
it should be listed in both tables.
Would you please check if the added features are only supported in mobile?
2. Also, doesn't these features need to be added to .NET feature page?

-> They are needed and I missed it. So I added them.

**Review by hyotaekshim — APPROVED** [review]
Look good

File: docs/application/native/tutorials/details/app-filtering.md [hunk 3]
```diff
@@ -281,10 +281,12 @@ package:
 | `http://tizen.org/feature/sensor.activity_recognition` | Specify this key, if the application requires an activity recognition sensor. | 2.3   |
```
[the remark this answers is not in the document]
Comment (lavataste) [reply docs/application/native/tutorials/details/app-filtering.md:281]: Isn't the "used" is for 'geomagnetic rotation vector sensor output", not "geomagnetic orientation sensor output"?
-> for rotation vector, so I did rollback the expressions "and is used" to "used"

Would you please check if the added features are only supported in mobile?
Also, doesn't these features need to be added to .NET feature page?

-> They are needed and I missed them. So I added them.

File: docs/application/dotnet/guides/sensors/device-sensors.md [hunk 4]
```diff
@@ -1,31 +1,36 @@
...
 |----------------------------------------|----------------------------------------|-------------------------------------|
-| [Accelerometer](#accelerometer)  | [Heart Rate Monitor LED Green Batch Sensor](#hrm_green_batch) | [Proximity Sensor](#proximity)   |
-| [AutoRotation Sensor](#auto_rotation) | [Humidity Sensor](#humidity) | [Rotation Vector Sensor](#rotation)  |
-| [Geomagnetic Rotation Vector Sensor](#mag_rotation) | [Light Sensor](#light) | [Sleep Monitor](#sleep_monitor)  |
-| [Gravity Sensor](#gravity)   | [Linear Acceleration Sensor](#lin_accelerometer) | [Temperature Sensor](#temperature)  |
-| [Gyroscope](#gyro)    | [Magnetic Sensor](#magnetic) | [Ultraviolet Sensor](#ultraviolet)  |
-| [Gyroscope Rotation Vector Sensor](#gyro_rotation)  | [Orientation Sensor](#orientation) | [Uncalibrated Gyroscope](#uncal_gyro)  |
-| [Heart Rate Monitor Sensor](#hrm)   | [Pedometer](#pedometer) | [Uncalibrated Magnetic Sensor](#uncal_magnetic) |
-| [Heart Rate Monitor Batch Sensor](#hrm_batch) | [Pressure Sensor](#pressure)  |  |
+| [Accelerometer](#accelerometer)  | [Heart Rate Monitor Batch Sensor](#hrm_batch) | [Proximity Sensor](#proximity)   |
```
Comment (arisha-srbd) [inline docs/application/dotnet/guides/sensors/device-sensors.md:25]: All the terms used here are using title-style capitalization. But in the file: docs/application/native/guides/location-sensors/device-sensors.md, all the terms are using sentence style capitalization. @mijin85cho requesting your input on this.

**Review by arisha-srbd — COMMENTED** [review]
Hello @lavataste,
Thank you for addressing the previous suggestions. We conducted another review on the files and we found some additional issues that were not addressed before. Along with that we have asked to revert back some previously misidentified problems. We apologize for the inconvenience. 

I am listing the issues and their correction suggestions below: 

**File: docs/application/dotnet/get-started/application-filtering.md**

Line 5: “Feature-based Filtering”
Suggestion: “Feature-based filtering”
Please use sentence style capitalization for second headings. Ref: https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#capitalization

Line 25: “Screen Size Feature”
Suggestion: “Screen size feature” 
Please use sentence style capitalization for second headings.

Line 50: “Feature Hierarchy”
Suggestion: “Feature hierarchy”
Please use sentence style capitalization for second headings.

Line 55: “application on the Tizen Store application list.”
Please check: Line appears to be broken in markdown but seems fine in staging area.

Line 59: This means that Tizen Store considers the `http://tizen.org/feature/location` feature as the `http://tizen.org/feature/location.gps OR http://tizen.org/feature/location.wps` feature.
Suggestion: This means that Tizen Store considers the `http://tizen.org/feature/location` feature as the `http://tizen.org/feature/location.gps’ or ‘http://tizen.org/feature/location.wps` feature.
I think the ‘or’ should be separated from the two hyperlinks and doesn’t require to be highlighted.  

Line 61: “Adding the Feature List”
Suggestion: “Add the feature list”
Please use sentence style capitalization for second headings and note the gerund case here. Ref: https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#capitalization
https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#verb-form

Line 65: To open the manifest editor in the Visual Studio, double-click the `tizen-manifest.xml` file in the **Solution Explorer** view.
Suggestion: To open the manifest editor in Visual Studio, double-click the `tizen-manifest.xml` file in the **Solution Explorer** view.
For nouns such as tool names, it is recommended to not use "The". 

Line 111 - “| `http://tizen.org/feature/location.fused` | Specify this key, if the application requires the Fused Location feature using sensors. | 4.0 |”
Suggestion – “| `http://tizen.org/feature/location.fused` | Specify this key, if the application requires the fused location feature using sensors. | 4.0 |”
Reference – Can you please explain why “Fused Location” has been capitalized here?

Line 108 - “| `http://tizen.org/feature/led` | Specify this key, if the application requires a LED.”
Suggestion – “| `http://tizen.org/feature/led` | Specify this key, if the application requires an LED.”
Since LED is pronounced here as "ell ee dee," I think it is better to use the article ‘an’ before it.

Line 247: “Profile-based Filtering”
Suggestion: “Profile-based filtering”
Please use sentence style capitalization for second headings.

**File: docs/application/native/guides/location-sensors/device-sensors.md**

Line 59: “If the `/<sensor-name>` element is omitted, the URI denotes any sensors of the given type, and therefore can correspond to more than 1 sensor.”
Suggestion: “If the `/<sensor-name>` element is omitted, the URI denotes any sensors of the given type and therefore can correspond to more than 1 sensor.”
The comma before “type” is not necessary here. Can you please check this? 

_Line 92, 93, 99 & 100, 159, 169, 177, 183, 216, 224 & 240-250:_ The “semi-colon” ending the codes aren’t consistent with the previous code snippet.
Can you please check this?

Line 152 - “…. For example, the proximity sensor emits its data only if the proximity value actually changes….”
Suggestion: “…. For example, the proximity sensor emits its data only if the proximity value changes.”
‘Only’ is already stressing on the meaning of the sentence. Hence, ‘actually’ is not required here. 

Line 172: “…However, it does not prevent the device from going to the sleep mode.”
Suggestion: “However, it does not prevent the device from going to sleep mode.”
Please refer to the article usage problem here. We can eliminate ‘the’ before ‘sleep mode’ here.
 
Line 197: “/* Pedometer is not supported or cannot be recorded on the current device */”
Suggestion: Since the above line is a comment within the code shouldn’t it be displayed in a lighter shade? [This issue was not addressed.] 

Line 227: “However, remember that the recording is not actually stopped, if another application has requested for the same sensor to be recorded.”
Suggestion: “However, if another application has requested for the same sensor data, remember that the initial recording is not stopped.”
_Requesting clarification – Based on our understanding, it seems you want to convey the message that when a new application requests to record the sensor data, the initial application that was recording the data is not actually stopped. So, the suggestion I have made has been derived from that context. Please check and make sure if it’s correct or not?_

Line 237 - “In the following example, you get an aggregated data record per day (since `SENSOR_RECORDER_QUERY_TIME_INTERVAL` is set to 24 hours) and a day starts at 7 AM (not at midnight, since `SENSOR_RECORDER_QUERY_ANCHOR_TIME` is set to 7 AM).”
Suggestion – “In the following example, you get an aggregated data record per day (since `SENSOR_RECORDER_QUERY_TIME_INTERVAL` is set to 24 hours) and a day starts at 7 AM (not at midnight, since `SENSOR_RECORDER_QUERY_ANCHOR_TIME` is set to 7 AM):”
Code snippets should be preceded by a “colon.” Please update this. 

Line 386 - “The rotation angles obtained by integrating the angular rates over longer duration is inaccurate due to the build-up of drift.”
Suggestion – “The rotation angles obtained by integrating the angular rates over a longer duration are inaccurate due to the build-up of drift.”
I would suggest using the article “a” before “longer duration” here. And it seems that the verb ‘is’ does not agree with the subject. 

Line 588 - “… The magnetic sensor measures the Earth's magnetic field strength and fluctuations, and splits the measurement into X, Y, and Z components.”
Suggestion – “The magnetic sensor measures the Earth's magnetic field strength and fluctuations and splits the measurement into X, Y, and Z components.”
The “comma” after the word ‘fluctuations’ is not necessary here.

Line 685 - “… When the device user holds the device to their ear, the proximity sensor detects the user as an object, and automatically locks the device screen.”
Suggestion – “When the device user holds the device to their ear, the proximity sensor detects the user as an object and automatically locks the device screen.”
The “comma” after the word ‘object’ is not necessary here.

Line 777 - “The rotation angles obtained by integrating the angular rates over longer duration is inaccurate due to the build-up of drift.”
Suggestion – “The rotation angles obtained by integrating the angular rates over a longer duration are inaccurate due to the build-up of drift.”
I would suggest using the article “a” before “longer duration” here. And it seems that the verb ‘is’ does not agree with the subject. 

Line 796 - “…. The uncalibrated magnetic sensor is similar in functionality to a magnetic sensor, but does not perform hard iron calibration.”
Suggestion – “The uncalibrated magnetic sensor is similar in functionality to a magnetic sensor but does not perform hard iron calibration.”
The “comma” after the word ‘sensor’ is not necessary here.

**File: docs/application/native/tutorials/details/app-filtering.md**

Line 46-51 - “The store only lists the applications whose conditions match the capabilities of the device, and thus prevents incompatible applications from being installed.”
Suggestion – “The store only lists the applications whose conditions match the capabilities of the device and thus prevents incompatible applications from being installed.”
The “comma” after the word ‘device’ is not necessary here.

Line 343-345 - “If you want to verify which features are necessary for using a specific API, see the related feature in the native [API Reference](../../api/overview.md).”
Suggestion – “If you want to verify which features are necessary for using a specific API, see the related feature in the Native [API Reference](../../api/overview.md).”
Since API names are common compound nouns, the word ‘Native’ should be capitalized. Ref: https://github.com/Samsung/tizen-docs/blob/master/styleguide/naming-rules.md#proper-nouns

File: docs/application/dotnet/guides/sensors/device-sensors.md (hunk 4 above)
Comment (mijin85cho) [reply docs/application/dotnet/guides/sensors/device-sensors.md:25]: After device-sensor pages for Native and .NET were set, we decided to use the sentence style to the titles.
Native had updates after the decision, so the sentence style was applied. 
So .NET is still in title-style, but Native is in sentence-style.

Comment (lavataste) [reply docs/application/dotnet/guides/sensors/device-sensors.md]: I changed to use sentence-style in .Net page

**Comment by lavataste** [comment]
@arisha-srbd

Thanks for your review.
Most of your sugesstions are applied.


Line 92, 93, 99 & 100, 159, 169, 177, 183, 216, 224 & 240-250: The ?semi-colon? ending the codes aren?t consistent with the previous code snippet.
Can you please check this?

Line 197: ?/* Pedometer is not supported or cannot be recorded on the current device */?
Suggestion: Since the above line is a comment within the code shouldn?t it be displayed in a lighter shade? [This issue was not addressed.]

-> In md file and github rendering, there is no problem but in Tizen site, the problems exist that Some characters had different brightness.
In source file, there are no difference way in using characters with other lines.
Do you know why this problem happen?




Line 227: "However, remember that the recording is not actually stopped, if another application has requested for the same sensor to be recorded."
Suggestion: "However, if another application has requested for the same sensor data, remember that the initial recording is not stopped."

->
In this situation, the recording keep going inner but it is not used in initial application, so I think below expression is proper.
"However, if another application has requested for the same sensor data, remember that the recording is not stopped."

File: docs/application/dotnet/guides/sensors/device-sensors.md [hunk 5]
```diff
@@ -287,59 +340,59 @@ The following table lists the measurement data that the gyroscope rotation vecto
...
-| HeartRate   | `int`                | Min. value = 0<br>Max. value = 240 | -            |
+| Measurement | Type       | Range                              | Unit         |
+|-------------|------------|------------------------------------|--------------|
+| Timestamp   | `ulong`    | -                                  | Microseconds |
+| TimeSpan    | `TimeSpan` | -                                  | Microseconds |
+| Heartrate   | `int`      | Min. value = 0<br>Max. value = 240 | -            |
 
 <a name="hrm_batch"></a>
-## Heart Rate Monitor Batch Sensor
+## Heart rate monitor batch Sensor
```
Comment (arisha-srbd) [inline docs/application/dotnet/guides/sensors/device-sensors.md:358]: Please use sentence-style capitalization. 
```suggestion
## Heart rate monitor batch sensor
```

**Review by arisha-srbd — COMMENTED** [review]
@lavataste thank you for addressing all the previous issues. We found a few minor ones remaining, please incorporate the suggestions we made on those. 

**File - docs/application/native/guides/location-sensors/device-sensors.md**

1.	Line 198 - “/* Pedometer is not supported or cannot be recorded on the current device */”
Issue: The comment in the code is appearing to be the same color as the code. (It should be greyed out). The problem was acknowledged by @lavataste but we can't seem to find the reason behind the issue. 

2.	Line 92, 93, 99 & 100, 159, 169, 177, 183, 216, 224 & 240-250: The “semi-colon" color ending the codes aren’t consistent with the previous code snippet.
Issue: We can't seem to find the reason behind this issue as well. 

@mijin85cho we will appreciate your input on this.

File: docs/application/dotnet/get-started/application-filtering.md [hunk 6]
```diff
@@ -47,22 +47,21 @@ The following table lists the available screen size features.
...
 
-### Feature Hierarchy
+### Feature hierarchy
 
 The feature keys have a hierarchy. For example, consider the `http://tizen.org/feature/location`, `http://tizen.org/feature/location.gps`, and `http://tizen.org/feature/location.wps` features:
 
--   If the feature list includes the `http://tizen.org/feature/location.gps` feature, only a device which has the `http://tizen.org/feature/location.gps` feature can show the
-    application on the Tizen Store application list.
+-   If the feature list includes the `http://tizen.org/feature/location.gps` feature, only a device which has the `http://tizen.org/feature/location.gps` feature can show the application on the Tizen Store application list.
```
[the remark this answers is not in the document]
Comment (lavataste) [reply docs/application/dotnet/get-started/application-filtering.md:54]: done

File: docs/application/dotnet/get-started/application-filtering.md [hunk 7]
```diff
@@ -47,22 +47,21 @@ The following table lists the available screen size features.
...
 
 - If the feature list includes the `http://tizen.org/feature/location` feature, a device which has the `http://tizen.org/feature/location.gps`, `http://tizen.org/feature/location.wps`, or `http://tizen.org/feature/location` feature can show the application on the Tizen Store application list.
```
[the remark this answers is not in the document]
Comment (lavataste) [reply docs/application/dotnet/get-started/application-filtering.md:56]: done

File: docs/application/dotnet/get-started/application-filtering.md [hunk 8]
```diff
@@ -47,22 +47,21 @@ The following table lists the available screen size features.
...
 
     This means that Tizen Store considers the `http://tizen.org/feature/location` feature as the `http://tizen.org/feature/location.gps OR http://tizen.org/feature/location.wps` feature.  (If the feature list includes the `http://tizen.org/feature/location.gps` and `http://tizen.org/feature/location.wps` features together, only a device which supports both those features can show the application.)
 
-### Adding the Feature List
+### Add the Feature List
```
[the remark this answers is not in the document]
Comment (lavataste) [reply docs/application/dotnet/get-started/application-filtering.md:60]: done

File: docs/application/dotnet/get-started/application-filtering.md [hunk 9]
```diff
@@ -47,22 +47,21 @@ The following table lists the available screen size features.
...
 
--   If the feature list includes the `http://tizen.org/feature/location.gps` feature, only a device which has the `http://tizen.org/feature/location.gps` feature can show the
-    application on the Tizen Store application list.
+-   If the feature list includes the `http://tizen.org/feature/location.gps` feature, only a device which has the `http://tizen.org/feature/location.gps` feature can show the application on the Tizen Store application list.
 
 - If the feature list includes the `http://tizen.org/feature/location` feature, a device which has the `http://tizen.org/feature/location.gps`, `http://tizen.org/feature/location.wps`, or `http://tizen.org/feature/location` feature can show the application on the Tizen Store application list.
 
     This means that Tizen Store considers the `http://tizen.org/feature/location` feature as the `http://tizen.org/feature/location.gps OR http://tizen.org/feature/location.wps` feature.  (If the feature list includes the `http://tizen.org/feature/location.gps` and `http://tizen.org/feature/location.wps` features together, only a device which supports both those features can show the application.)
```
[the remark this answers is not in the document]
Comment (lavataste) [reply docs/application/dotnet/get-started/application-filtering.md:58]: done

File: docs/application/dotnet/guides/sensors/device-sensors.md (hunk 5 above)
Comment (lavataste) [reply docs/application/dotnet/guides/sensors/device-sensors.md:358]: done

**Comment by mijin85cho** [comment]
> @lavataste thank you for addressing all the previous issues. We found a few minor ones remaining, please incorporate the suggestions we made on those.
> 
> **File - docs/application/native/guides/location-sensors/device-sensors.md**
> 
> 1. Line 198 - “/* Pedometer is not supported or cannot be recorded on the current device */”
>    Issue: The comment in the code is appearing to be the same color as the code. (It should be greyed out). The problem was acknowledged by @lavataste but we can't seem to find the reason behind the issue.
> 2. Line 92, 93, 99 & 100, 159, 169, 177, 183, 216, 224 & 240-250: The “semi-colon" color ending the codes aren’t consistent with the previous code snippet.
>    Issue: We can't seem to find the reason behind this issue as well.
> 
> @mijin85cho we will appreciate your input on this.

I think it is language identifier.

**Reference :** https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#inline-code-blocks-with-language-identifier


In page native/guides/location-sensors/device-sensors.md, the code block was made without language identifier.
image

File: docs/application/dotnet/get-started/application-filtering.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (arisha-srbd) [reply docs/application/dotnet/get-started/application-filtering.md:56]: @lavataste could you please check the error again? It seems like the article 'the' was not removed before 'Tizen Store' here.

File: docs/application/dotnet/get-started/application-filtering.md [hunk 10]
```diff
@@ -47,22 +47,21 @@ The following table lists the available screen size features.
...
 
-### Feature Hierarchy
+### Feature hierarchy
 
 The feature keys have a hierarchy. For example, consider the `http://tizen.org/feature/location`, `http://tizen.org/feature/location.gps`, and `http://tizen.org/feature/location.wps` features:
 
--   If the feature list includes the `http://tizen.org/feature/location.gps` feature, only a device which has the `http://tizen.org/feature/location.gps` feature can show the
-    application on the Tizen Store application list.
+-   If the feature list includes the `http://tizen.org/feature/location.gps` feature, only a device which has the `http://tizen.org/feature/location.gps` feature can show the application on Tizen Store application list.
```
Comment (arisha-srbd) [inline docs/application/dotnet/get-started/application-filtering.md:54]: Minor correction to address a grammatical issue here.
```suggestion
-   If the feature list includes the `http://tizen.org/feature/location.gps` feature, only a device which has the `http://tizen.org/feature/location.gps` feature can show the application on Tizen Store's application list.
```

**Review by arisha-srbd — COMMENTED** [review]
@lavataste hope you are doing well. Thank you for incorporating all the changes. 

Just a few concerns that we had before can now be addressed I think. According to @mijin85cho's comment: https://github.com/Samsung/tizen-docs/pull/1585#issuecomment-1053988266, please note that incorrect colors in the code could be because of lack of language identifiers within the code. We would appreciate it if you could add the identifiers in the specific places to get rid of this issue.

File: docs/application/dotnet/get-started/application-filtering.md (hunk 10 above)
Comment (lavataste) [reply docs/application/dotnet/get-started/application-filtering.md]: Thank you. Your suggestion was applied.

**Review by arisha-srbd — COMMENTED** [review]
@lavataste thank you for addressing the previously suggested changes. We gave another check and found a small inconsistency between the paragraph and the table in the .NET file when compared to the native file. I am listing the problems below: 

**File - docs/application/dotnet/guides/sensors/device-sensors.md**

_We noticed that there are 5 values labeled in the tables below the paragraphs, and so we are suggesting to change the number of values to 5 and include the missing parameter. Can you please check this?_ 

1.	Line 165 – “The accelerometer outputs 4 values: 3 Cartesian axis values and a timespan. The accelerometer sensor measures and returns the axes’ values in “m/s2” (meters per second squared). When a device is moved in the ±X, ±Y, or ±Z direction, the corresponding output increases (+) or decreases (-).”
_Suggestion_ – “The accelerometer outputs 5 values: 3 Cartesian axis values, a timestamp, and a timespan. The accelerometer sensor measures and returns the axes’ values in “m/s2” (meters per second squared). When a device is moved in the ±X, ±Y, or ±Z direction, the corresponding output increases (+) or decreases (-).”
Reason – 5 values are mentioned in the table below.

2.	Line 263 – “The gravity sensor outputs 4 values: 3 Cartesian axis values and a timespan. The gravity sensor measures and returns axes values in “m/s2” (meters per second squared). When a device is rotated in the ±X, ±Y, or ±Z direction, the corresponding output increases (+) or decreases (-).”
_Suggestion_ – “The gravity sensor outputs 5 values: 3 Cartesian axis values, a timestamp, and a timespan. The gravity sensor measures and returns axes values in “m/s2” (meters per second squared). When a device is rotated in the ±X, ±Y, or ±Z direction, the corresponding output increases (+) or decreases (-).”
Reason – 5 values are mentioned in the table below.

3.	Line 455 – “The linear acceleration sensor outputs 4 values: 3 Cartesian axis values and a timespan. The linear acceleration sensor measures and returns axes values in “m/s2” (meters per second squared). When a device is accelerated in the ±X, ±Y, or ±Z direction, the corresponding output increases (+) or decreases (-). The acceleration output is shown in the same direction as the user-driven force.”
_Suggestion_ – “The linear acceleration sensor outputs 5 values: 3 Cartesian axis values, a timestamp, and a timespan. The linear acceleration sensor measures and returns axes values in “m/s2” (meters per second squared). When a device is accelerated in the ±X, ±Y, or ±Z direction, the corresponding output increases (+) or decreases (-). The acceleration output is shown in the same direction as the user-driven force.”
Reason – 5 values are mentioned in the table below.

**Comment by lavataste** [comment]
@arisha-srbd

**Comment by lavataste** [comment]
@arisha-srbd 
Thanks for you review, your suggestions were applied.

**Review by arisha-srbd — APPROVED** [review]
@lavataste thank you for incorporating all the changes. 

@asm-foysal @mijin85cho I am approving this PR to be merged.

**Pull request merged**